src/app/component/core/header-new/header-new.component.ts

```diff
@@ -14,22 +13,24 @@ import { HabitStatisticService } from 'src/app/service/habit-statistic/habit-sta
 import { filter } from 'rxjs/operators';
 import { LanguageService } from '../../../i18n/language.service';
 import { Language } from '../../../i18n/Language';
+import {SignInNewComponent} from '../../auth/sign-in-new/sign-in-new.component';
+import {NewSignUpComponent} from '../../auth/new-sign-up/new-sign-up.component';
```

Comment:
General: add space between brackets in imports

Reply:
+